src/averageAges.js

```diff
@@ -37,7 +42,16 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const women = withChildren
+    ? people.filter((p) => p.sex === 'f'
```

Comment:
Maybe take into consideration to NOT name variables as a single letters. Might be useful especially at the beginning to make it more readable (especially for yourself)